.github/workflows/codespace-test.yml

```
@@ -0,0 +1,40 @@
+        DJANGO_DEBUG: 'True'
+        DJANGO_ALLOWED_HOSTS: 'localhost,127.0.0.1'
+      run: |
+        python manage.py migrate
+    
+    - name: Run tests
+      env:
+        DJANGO_SECRET_KEY: 'test-secret-key'
+        DJANGO_DEBUG: 'True'
+        DJANGO_ALLOWED_HOSTS: 'localhost,127.0.0.1'
+      run: |
+        python manage.py test account --settings=core.test_settings -v 2
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)